src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/LLVMCodegenCompilationBuilder.cs

```diff
@@ -72,7 +77,9 @@ internal void SetOptions(IEnumerable<KeyValuePair<string, string>> options)
         // S128 natural alignment of stack
         public string DataLayout { get; private set; } = "e-m:e-p:32:32-i64:64-n32:64-S128";
         public string Target { get; private set; } = "wasm32-unknown-emscripten";
-        public string ModuleName { get; private set; } = "netscripten";
         public CorInfoLlvmEHModel ExceptionHandlingModel { get; private set; } = CorInfoLlvmEHModel.Cpp;
+
+        // Below options are debug-only and not supported.
+        public int MaxLlvmModuleCount { get; private set; } = 8;
```

Comment:
This is the hardcoded number. Ideally, it would be correlated with how "large" the input is instead.

`8` is a conservative number, but this is also an incomplete solution, so there is no great point in tuning it now.